client/__tests__/e2e/e2e.test.ts

```diff
@@ -1620,6 +1763,7 @@ for (const testDataset of testDatasets) {
 
           test("with continuous legend", async ({ page }, testInfo) => {
             await goToPage(page, url);
+            await showImageUnderlayInTestMode(page);
```

Comment:
should this be in snapshotTestGraph?

Reply:
Great question! There are cases we want to take a snapshot of the graph itself but not the image underlay, so that's why the two functions are separated 🙆‍♂️ Hope that addresses your question!